boot/bootstate20_bloader_kernel_state.go

```diff
@@ -177,6 +177,26 @@ func (bks *extractedRunKernelImageBootloaderKernelState) setNextKernel(sn snap.P
 	return nil
 }
 
+func (bks *extractedRunKernelImageBootloaderKernelState) setNextKernelNoTry(sn snap.PlaceInfo) error {
+	if sn.Filename() != bks.currentKernel.Filename() {
+		err := bks.ebl.EnableKernel(sn)
+		if err != nil {
+			return err
+		}
+	}
+
+	if bks.currentKernelStatus != "" {
+		m := map[string]string{
+			"kernel_status": "",
+		}
```

Comment:
nitpick: I guess you can write `DefaultStatus` instead of ""`.

Reply:
Fixed now